.ci/setup_env_ubuntu.sh

```diff
@@ -49,7 +49,11 @@ if [ "${NAME}" == "Ubuntu" ] && [ "$(echo "${VERSION_ID} >= 20.04" | bc -q)" ==
 fi
 
 if [ "$(uname -m)" == "x86_64" ] && [ "${NAME}" == "Ubuntu" ] && [ "$(echo "${VERSION_ID} >= 18.04" | bc -q)" == "1" ]; then
-	packages[qemu_dependencies]+=" libpmem-dev"
+	packages[qemu_dependencies]+="libpmem-dev"
```

Comment:
We need a space before the `libpmem-dev`

Reply:
Updated, sorry!